utils.py

```diff
@@ -37,8 +37,13 @@
 import feconf
 import python_utils
 
+from typing import ( # isort:skip # pylint: disable=unused-import, import-only-modules
+    Any, Callable, cast, Dict, Generator, Iterable, Iterator, List, # isort:skip # pylint: disable=unused-import, import-only-modules
+    Optional, Text, Tuple, TypeVar, Union) # isort:skip # pylint: disable=unused-import, import-only-modules
```

Comment:
How would we ensure that all the imports are used in type?

Reply:
There is no way to do that as everything imported here is used in type comments.

Reply:
I think we can write a pylint check to ensure all the imports are used in the file, right? (Though this can be a future work)

Reply:
We will be changing the type annotation syntax after python3 migration. 
The future annotations will look like this(after py3 migration):
`def(x: int, y: int) -> int:`
Do you think linters will still find the imported values unused?

Reply:
Do we have any close expected date for the python3 migration completion? If not, I would suggest creating an issue to have a linter check and we can add one later!

Reply:
https://github.com/oppia/oppia/issues/13169 Added1

Reply:
> Do you think linters will still find the imported values unused?

I'm not sure but until the python3 migration is not completed we have a lint check to ensue that we don't have any unwanted typing imports in a file*

Reply:
I don't think we should put in time here. Py3 migration will be complete in a month or so. Till then manual checking of imports would be the best option. Thanks!

Reply:
@DubeySandeep . Python3 migration will be completed in a month or so. I don't think this is required. Thanks!